llama_hub/elasticsearch/base.py

```diff
@@ -4,11 +4,11 @@
 
 """
 
-
 from typing import List, Optional
 
 from llama_index.readers.base import BaseReader
 from llama_index.readers.schema.base import Document
+from elasticsearch import Elasticsearch
```

Comment:
let's do lazy imports (for tests to pass)

Reply:
It's done!